data/alphabet.txt

```diff
@@ -0,0 +1,32 @@
+# Each line in this file represents the text (usually a single character, but
+# not required) associated with a numeric label.
+# A line that starts with # is a comment.
```

Comment:
Probably not very likely as that would screw up with the acoustic modeling, that case should probably be normalized to "a number two pencil". We don't currently have any code to do that because the datasets themselves handle it, but in any case, I'll add support for escaping the initial #.